oaknational/Oak-Web-Application
# Fix/404 get attempt bug

## Description

Music year: 1989

no ticket

we were getting a large amount of 404s for get requests to the api.

we noticed if you go to printable page with any attemp_id it makes 404 requests to api

now the printable page doesnt make calls to api (it was a bug)
 
the printable results button is now a anchor tag; and the lesson is stored in local storage and the url is created apon lesson completion

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2848--oak-web-application.netlify.thenational.academy
2. Click on a lesson; complete the lesson
3. You should see that the lesson attempt is stored in local storage one; and the printable lesson button takes you to printable results page; printable results button is an anchor tag

1. Go to https://deploy-preview-2848--oak-web-application.netlify.thenational.academy/pupils/lessons/what-is-representation-ccrpar/results/any-id/printable
3. You should see that no get attemots are made to the api




## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
